js/background.js

```diff
@@ -483,40 +499,45 @@
         });
     }
 
-    function onMessageReceived(ev) {
-        var data = ev.data;
-        if (data.message.flags & textsecure.protobuf.DataMessage.Flags.PROFILE_KEY_UPDATE) {
-            var profileKey = data.message.profileKey.toArrayBuffer();
-            return ConversationController.getOrCreateAndWait(data.source, 'private').then(function(sender) {
-              return sender.setProfileKey(profileKey).then(ev.confirm);
-            });
-        }
-        var message = initIncomingMessage(data);
-
-        return isMessageDuplicate(message).then(function(isDuplicate) {
-            if (isDuplicate) {
-                console.log('Received duplicate message', message.idForLogging());
-                ev.confirm();
-                return;
-            }
+  /* eslint-enable */
+  /* jshint ignore:start */
+  async function onMessageReceived(ev) {
+    const { data } = ev;
```

Comment:
I notice that here you put commas inside your `{}` where in most other places you don't do that. For the record, I like the spaces. Our current eslint config is AirBnB with just a few tweaks.

Reply:
Re: commas. Did you mean spaces? Yes, I am happy to do either and just followed what the linter told me. If I missed any, let me know and I’ll enable linting there 😄